orcid-core/src/main/resources/i18n/javascript_xx.properties

```diff
@@ -151,3 +151,4 @@ orcid.frontend.workspace.your_primary_email=X
 orcid.frontend.workspace.youve_changed=X
 submitButton.label=X
 workspace.works.import_wizzard.all=X
+public-layout.trusted_individuals=X
```

Comment:
Could you remove this property since it's already in the `messages_en.properties`